system/netdata.service.in

```diff
@@ -17,7 +17,7 @@ ExecStartPre=/bin/mkdir -p @localstatedir_POST@/cache/netdata
 ExecStartPre=/bin/chown -R netdata:netdata @localstatedir_POST@/cache/netdata
 ExecStartPre=/bin/mkdir -p @localstatedir_POST@/run/netdata
 ExecStartPre=/bin/chown -R netdata:netdata @localstatedir_POST@/run/netdata
-#ExecStopPost=/bin/rm @localstatedir_POST@/run/netdata/netdata.pid
+ExecStopPost=/usr/libexec/netdata/plugins.d/reset_netdata_trace.sh
```

Comment:
`plugins.d` can be installed to a different directory.

Reply:
I added the missing prefix, thank you!